.github/workflows/main.yml

```diff
@@ -112,6 +112,10 @@ jobs:
       if: env.IS_CD == 'false'
       run: ./gradlew test --tests='*ExternalTest'
```

Comment:
@sgrimm I can't seem to get the ExternalTest to work. Even though I can run the tests successfully if I set the env variables locally.